test/test_functional_tensor.py

```diff
@@ -578,6 +579,52 @@ def test_assert_resize_antialias(interpolation):
         F.resize(tensor, size=(5, 5), interpolation=interpolation, antialias=True)
 
 
+@pytest.mark.parametrize('dt', [torch.float32, torch.float64, torch.float16])
+@pytest.mark.parametrize('size', [[10, 7], [10, 42], [42, 7]])
+@pytest.mark.parametrize('interpolation', [BILINEAR, BICUBIC])
+def test_interpolate_antialias_backward(dt, size, interpolation):
+
+    # temporarily hard-code device as CPU, CUDA support will be done later
+    device = "cpu"
+
+    if dt == torch.float16 and device == "cpu":
+        # skip float16 on CPU case
+        return
+
+    torch.manual_seed(12)
+    if interpolation == BILINEAR:
+        forward_op = torch.ops.torchvision._interpolate_bilinear2d_aa
+        backward_op = torch.ops.torchvision._interpolate_bilinear2d_aa_backward
+    elif interpolation == BICUBIC:
+        forward_op = torch.ops.torchvision._interpolate_bicubic2d_aa
+        backward_op = torch.ops.torchvision._interpolate_bicubic2d_aa_backward
+
+    class F(torch.autograd.Function):
+
+        @staticmethod
+        def forward(ctx, i):
+            result = forward_op(i, size, False)
+            ctx.save_for_backward(i, result)
+            return result
+
+        @staticmethod
+        def backward(ctx, grad_output):
+            i, result = ctx.saved_tensors
+            ishape = i.shape
+            oshape = result.shape[2:]
+            return backward_op(grad_output, oshape, ishape, False)
```

Comment:
This is fine for now. I suppose the next step would be to move those functions to PyTorch so that we nave native autograd support?